src/involvement.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-console */
+import { renderLikes } from './render-likes';
+import commentsCounter from './comments-counter';
+
+const appId = 'emZFdA0IO3Wtnd0ZXf1j';
+
+export const postLike = async (idMeal) => {
+  await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/likes/`, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: idMeal,
+    }),
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+  })
+    .catch((error) => console.error(error));
+};
+
+export const getLikes = async () => {
+  await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/likes/`, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((res) => res.json())
+    .then((data) => renderLikes(data))
+    .catch((error) => console.error(error));
+};
+
+export const postComment = async (idMeal, username, comment) => {
+  await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/comments`, {
+    method: 'POST',
+    body: new URLSearchParams({ item_id: idMeal, username, comment }),
+  })
+    .then((res) => res.text())
+    .then((text) => console.info(text));
```

Comment:
- [ ]  Please don't use `console` output. The JavaScript is designed to be executed in the browser, it's considered a best practice to avoid using methods on console. Such messages are considered to be for debugging purposes and therefore not suitable to be in public projects.
Please check  [this](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20console.log%20to%20your%20repo.%20It%E2%80%99s%20ugly%2C%20it%20kills%20performance%20and%20it%20can%20make%20confidential%20data%20be%20visible%20to%20anyone%20using%20the%20browser%20tools%20to%20look%20at%20your%20website.) best practices guide for more info